### PR TITLE
Issues/10 base html architecture

### PR DESCRIPTION
**Pull request by acecconato**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="robots" content="all">
+    <meta name="title" content="Reservia, trouvez votre hébergement de reve">
+    <meta name="description"
+          content="Ut De erga ferri amicis facit autem fiat De pariter nos erga nosmet ut ipse in aequaliterque se tertiam benevolentiae video adfecti ut quanti ut nullam nosmet in ut eodem.">
+    <link rel="stylesheet" href="assets/css/main.css">
```
Comment (Tutt-J) [inline index.html:12]: It does'nt make sense to add link to stylesheet heure. We only want meta.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,290 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="robots" content="all">
+    <meta name="title" content="Reservia, trouvez votre hébergement de reve">
+    <meta name="description"
```
Comment (Tutt-J) [inline index.html:10]: Please wrote a real description like "Reservia, l'hébergement de vos rêves à porter de main."

File: index.html [hunk 3]
```diff
@@ -0,0 +1,290 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="robots" content="all">
+    <meta name="title" content="Reservia, trouvez votre hébergement de reve">
```
Comment (Tutt-J) [inline index.html:9]: Title must be wrote like this :       <title>Reservia</title>

File: index.html [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+    <meta name="description"
+          content="Ut De erga ferri amicis facit autem fiat De pariter nos erga nosmet ut ipse in aequaliterque se tertiam benevolentiae video adfecti ut quanti ut nullam nosmet in ut eodem.">
+    <link rel="stylesheet" href="assets/css/main.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css"
+          integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+    <title>Reservia</title>
+</head>
+<body>
+
+<header>
```
Comment (Tutt-J) [inline index.html:19]: The header must be empty at this step !

File: index.html [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+        <ul>
+            <li><h1>Reservia</h1></li>
+            <li>S'enregistrer</li>
+            <li>Hébergements</li>
+            <li>Activités</li>
+        </ul>
+    </nav>
+</header>
+
+<section>
```
Comment (Tutt-J) [inline index.html:30]: No section at this step only body with emtpy header, empty main and empty footer

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (acecconato) [reply index.html:12]: Stylesheet link removed

File: index.html [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
```
Comment (Tutt-J) [inline index.html:2]: lang must be fr because our page is in french

**Review by Tutt-J — CHANGES_REQUESTED** [review]

**Pull request merged**